test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+Import-Module (Join-Path -Path $PSScriptRoot '..\Microsoft.PowerShell.Security\certificateCommon.psm1')
+Import-Module HelpersCommon
```

Comment:
Previously, we removed the explicit module loading and use autoloading.

Reply:
Is `HelpersCommon` path there in the module path var in the CI agent machine ?

Reply:
I believe HelpersCommon is loaded when Start-PSPester is called.

Reply:
Yep, `HelpersCommon` is loaded in `Start-PSPester`.

Reply:
Start-PSPester add path to test module directory in PSModulePath. So all test modules auto loaded.

Reply:
yup, but `certificateCommon` is not. @iSazonov, Correct me if I'm wrong.

Reply:
It is here:
https://github.com/PowerShell/PowerShell/blob/83708a4242befaae0e26d7f33e7ebe2f0091db08/build.psm1#L171-L174

Reply:
Modified.